Orange/widgets/utils/colorbrewer.py

```diff
@@ -1,3 +1,9 @@
+import warnings
+
+warnings.warn("Module 'colorbrewer' is obsolete and will be removed.\n"
+              "Use palettes from 'Orange.misccolorpalettes'.",
```

Comment:
`Orange.widget.utils.colorpalettes` ?

Reply:
Yes, this module travelled around a bit.